test/e2e/dra/test-driver/deploy/example/admin-access-policy.yaml

```diff
@@ -22,10 +22,14 @@ spec:
       apiVersions: ["v1alpha3", "v1beta1"]
       operations:  ["CREATE", "UPDATE"]
       resources:   ["resourceclaims"]
+    objectSelector:
+      matchExpressions:
+        - key: vap-enforced
+          operator: Exists
```

Comment:
I found that using a `namespaceSelector` here like @pohly tried in #128493 doesn't work because that selector operates only on labels, not other fields like the name. Open to suggestions if there's a better way to apply this policy less widely.

Reply:
So the idea is that this VAP only applies to claims and claim templates which opt into using it. I can see how this works for E2E testing.

But it's not something that admins should do for real, so this would need more comments along the lines of "don't do this when setting up admin access in your cluster".

Given that we probably will have a different enforcement mechanism for admin access (https://github.com/kubernetes/enhancements/issues/5018), let's perhaps put this PR on hold for now?

Reply:
/hold 
until https://github.com/kubernetes/enhancements/issues/5018 is implemented.